building/tracks/config-json.md

```diff
@@ -435,32 +461,32 @@ This is an example of what a valid `config.json` file can look like:
   ],
   "key_features": [
     {
-      "icon": "features-oop",
+      "icon": "expressive",
       "title": "Modern",
       "content": "C# is a modern, fast-evolving language."
     },
     {
-      "icon": "features-strongly-typed",
+      "icon": "statically-typed",
       "title": "Cross-platform",
       "content": "C# runs on almost any platform and chipset."
     },
     {
-      "icon": "features-functional",
+      "icon": "functional",
```

Comment:
```suggestion
      "icon": "multi-paradigm",
```

Assuming that we want to better match the below
```
      "title": "Multi-paradigm",
      "content": "C# is primarily an object-oriented language, but also has lots of functional features."
```

(unless the point was to show that we can choose the icon independently).